cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1584,9 +1584,10 @@ function Install_Containerd {
   $tmp_dir = 'C:\containerd_tmp'
   New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
 
-  # TODO(ibrahimab) Change this to a gcs bucket with CI maintained and accessible by community.
-  $version = '1.4.4'
-  $tar_url = ("https://github.com/containerd/containerd/releases/download/v${version}/" +
+  # TODO(ibrahimab) Change this to pick from a CI maintained and accessible by community.
+  # Current (1.5.2) version on the gke-release bucket is patched with PR pull/5411 on containerd repo.
```

Comment:
I'm not in favor of changing this to unofficial versions... why is this needed here?

Reply:
I'm fine, and slightly on the side of just using the github release bins here on the main repo. The only reason is that there is csi driver issues that haven't made it yet in any release, and this causes the master containerd grid to have those tests failing (https://testgrid.k8s.io/sig-windows-gce#gce-windows-2019-containerd-master) - though, I wanted to switch to the built binaries we use on gke-release. Also, we've multiple gce/gke components that gets extracted through the prod gcs bucket, like logging agents, npd, pigz, etc.. So, not sure if going forward we need to have all changed too ?!.

Reply:
@jeremyje , any thoughts here ?

Reply:
I had an offline discussion with @jeremyje. Since 1.5.3 is almost there, let's wait and keep the bin pointed to the public releases then. I'll send a PR when it's available to just change the used version.